Editorial pass on use of the word requires (#348)
w3c/wcag2ict: comments-on-definitions-in-wcag-2.2-glossary-in-appendix-a.md

Comments on Definitions in WCAG 2 Glossary
----------------------------------------------------------

> The following is a complete list of definitions from the WCAG 2 glossary. Some items apply to all technologies and do not need the additional guidance provided by this document; guidance on the remainder follows.

Reviewer: ```suggestion
The following is a complete list of definitions from the WCAG 2 glossary. Some items apply to all technologies and do not need to have additional guidance provided by this document; guidance on the remainder follows.
```

        
        **OR**
        

>     3.  The content is available in a closed environment, such as a university or corporate network, where the user agent <INS>**[or other software]**</INS> depends upon the technology used by the organization is also accessibility supported;

Reviewer: We aren't at liberty to change the normative WCAG definitions. This is the exact language contained in the definition of "accessibility supported" (other than the word substitution WCAG2ICT put in there.

<DL><DT>assistive technology (as used in this document)</DT><DD>

> hardware and/or software that acts <INS>**[stand-alone]**</INS>, or along with <INS>**[mainstream information and communication technologies (ICT)]**</INS>, to provide functionality to meet the needs of users with disabilities that go beyond those offered by <INS>**[mainstream ICT]**</INS>

Reviewer: ```suggestion
hardware and/or software that acts <INS>**[stand-alone]**</INS>, or along with <INS>**[mainstream information and communication technologies (ICT)]**</INS>, to provide functionality to meet the requirements of users with disabilities that go beyond those offered by <INS>**[mainstream ICT]**</INS>
```

<DL><DT>cognitive function test</DT><DD>

> A task that where the user needs to remember, manipulate, or transcribe information. Examples include, but are not limited to:

Reviewer: ```suggestion
A task that requires the user to remember, manipulate, or transcribe information. Examples include, but are not limited to:
```

<div class="note">This includes:

> 1.  Information that is needed by the <INS>**[[non-web document](#document) or [software](#software)]**</INS> but omitted by the user

Reviewer: ```suggestion
1.  Information that is required by the <INS>**[[non-web document](#document) or [software](#software)]**</INS> but omitted by the user
```

1.  Information that is required by the <INS>**[[non-web document](#document) or [software](#software)]**</INS> but omitted by the user
    

> 2.  Information that is provided by the user but that falls outside data format parameters or values

Reviewer: ```suggestion
2.  Information that is provided by the user but that falls outside the required data format or values
```
